### ISSUE TITLE
Error when there is no 'time' column (for labchart txt files)

### ISSUE THREAD

**Issue by RayStick** [physiopy/phys2bids#138]
When using phys2bids with a labchart txt file that does _not_ have time as the first column, the file is not correctly processed. 

## Expected Behavior

When I run:

`phys2bids -in Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.txt -info -chtrig 0`

I expect:

```
File extension is .txt
Reading the file
Reading infos
File Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.txt contains:

00. CO2; sampled at 1000.0 Hz
01. O2; sampled at 1000.0 Hz
02. Pulse; sampled at 1000.0 Hz
saving channels plot at plot at Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.png
```

I expect the plot to show: trigger, CO2, O2 and pulse. 

## Actual Behavior

This is what I get:

```
File extension is .txt
Reading the file
Reading infos
File Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.txt contains:

00. Trigger; sampled at 1000.0 Hz
01. CO2; sampled at 1000.0 Hz
saving channels plot at plot at Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.png
```

![Test2_trigger_CO2_O2_pulse_1000Hz_534TRs](https://user-images.githubusercontent.com/50215726/73800596-ba950080-477e-11ea-8a97-7cb68bfda859.png)

## Steps to Reproduce the Problem

Run the command I ran above. The data can be found on OSF/labchart/chicago
    
## Specifications

Python version: 3.7
phys2bids version: phys2bids v1.1.0-beta+236.g3a6fc72
Platform: MacOS

## Possible solution
I am not sure. It seems to be plotting time as the first column, even though I specified that trigger was '0'. Then I am unsure what the second and third are!

**Comment by vinferrer** [comment]
Oh I see, i didn't expected that labchart wouldn't give the time as first column, this will need a new pull request to solve this bug and probably add a new test

**Comment by eurunuela** [comment]
I'd make a new PR once #123 is merged. Of course, more unit tests will be needed.

**Comment by RayStick** [comment]
Well, the way our lab exports files as txt from LabChart, we _do_ export a time column.  However, there is an option not to export a time column, in the export.  So, for now, in the phys2bids guidance, we can say that it expects a time column (which isn't a big deal), but if it is a simple enough thing to change in future it'd be good to have the flexibility to read in labchart txt files without time column.

**Comment by eurunuela** [comment]
Is there a way you could send us (or upload) one of those files without the time column? It could come in handy to test the code.

**Comment by vinferrer** [comment]
Is already in osf.io

**Comment by smoia** [comment]
I would explain that for the moment files need to
1. have an header and
2. if in txt format, have a time column.

It might even be a good idea to add this in the help/ first messages of the program.

We can remove it once the feature is implemented.

**Comment by vinferrer** [comment]
sure but don't close the issue, I think it has an easy fix

**Comment by vinferrer** [comment]
Update on this, I have been playing with the file ``Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.txt`` is this what you expect the plot to look like?
![image](https://user-images.githubusercontent.com/38909338/74013783-89d6e600-498d-11ea-8368-8936e4b35764.png)

Here is the command that generated the image:
``phys2bids -in Test2_trigger_CO2_O2_pulse_1000Hz_534TRs.txt -info``

**Comment by vinferrer** [comment]
If so I will submit a PR to solve the problem

**Comment by RayStick** [comment]
Yes, that's the plot I expected! Great.
